src/components/Select/Select.tsx

```diff
@@ -0,0 +1,115 @@
+import * as React from "react";
+import bem from "../../utils/bem";
+import Icon from "../Icons/Icon";
+
+export interface SelectProps {
+    /** blockName for use with BEM. See how to work with modifiers and BEM here: http://getbem.com/introduction/ */
+    blockName?: string;
+
+    /** Modifiers array for use with BEM. See how to work with modifiers and BEM here: http://getbem.com/introduction/ */
+    modifiers?: string[];
+
+    /** ID that other components can cross reference for accessibility purposes */
+    id?: string;
+
+    /** ID of associated label */
+    labelId?: string;
+
+    /** ID of associated HelperText */
+    helperTextId?: string;
+
+    /** Attribute indicating that an option with a non-empty string value must be selected */
+    isRequired: boolean;
+
+    /** When passed, will populate the aria-label on the select */
+    ariaLabel?: string;
+
+    /** When true, disables the select */
+    disabled?: boolean;
+
+    /** Populates the selects' options */
+    options: string[];
+
+    /** Sets whatever string you pass it as the default selected */
+    selectedOption?: string;
+
+    onSelectBlur: (event: React.FormEvent) => void;
+
+    /** Passes selects' current value to the React state handler */
+    onSelectChange: (event: React.FormEvent) => void;
```

Comment:
My personal preference would be to name these `onBlur`, `onChange`, etc to keep the API as close to the HTML api as possible